kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/predicates/In.java

```diff
@@ -89,7 +89,7 @@ public boolean apply(@Nullable T test) {
     return new In<V>(Iterables.transform(set, function));
   }
 
-  Set<T> getSet() {
+  public Set<T> getSet() {
```

Comment:
Minor: there is also a method, `Predicates.asSet(In)`, that exposes the inner set. I think the purpose of that was to mimic the use of `Ranges.asSet(Range)`, which has to be a static method because we allow ranges of objects that aren't `Comparable` but back the `asSet` method with a Guava range that requires a `Comparable` type.

I'm fine with changing to use this method everywhere, since this is more straight-forward. But I'd like to either use `Predicates.asSet(In)` or `In#getSet()`, not both.